__integrationtests__/mergeDeployablePullRequests.test.ts

```diff
@@ -26,5 +26,6 @@ const setup = () => {
   const github = new GithubApiManager(auth, "deliveroo", "dev-glue");
   const workingDirectory = mkdtempSync("git-workspace");
   const git = new GitCommandManager(workingDirectory, GITHUB_USER, GITHUB_PAT);
+  // TODO: should we create PRs and such?
```

Comment:
I'm not a big fan of `TODO`s in committed codebases; would this be better as a ticket? It could have more context there, which would be good if we don't address this TODO soon :)

Reply:
this wasn't supposed to be committed :D